### PR TITLE
Refactor DistSQL drop scaling jobId to clean scaling jobId

### LINKED ISSUE #20275: Refactor DistSQL drop scaling jobId to clean scaling jobId
Current DistSQL: drop scaling j01001;
Final DistSQL: clean scaling j01001;

There're plenty of classes and files need to be modified, include grammar files and related classes, and also DistSQL handler, unit test etc.
We could find them out by searching:
1. Class name search by text: DropMigration . Rename classes, replace `Drop` to `Clean`
2. Text search by regex: drop[ _-]*scaling . Replace text `drop` to `clean` , classes generated by ANTLR4 could be ignored

Firstly, we could change ANTLR4 grammer file, replace `drop` to `clean`:
- migration/RALStatement.g4
- MigrationDistSQLStatement.g4

Then run `mvn clean install -Dmaven.javadoc.skip=true -Djacoco.skip=true -Dspotless.apply.skip=true` in `shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql` module.

And then continue to rename or replace by keyword searching.

After go through these changed files, we could basically know how to add or modify scaling DistSQL.

### PR DESCRIPTION
**Pull request by huangdx0726**
Fixes #20275 .

Changes proposed in this pull request:
-
-  replace drop to clean in some files
-

### CODE REVIEW

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/java/org/apache/shardingsphere/migration/distsql/handler/update/CleanMigrationUpdater.java [hunk 1]
```diff
@@ -20,22 +20,22 @@
 import org.apache.shardingsphere.data.pipeline.api.MigrationJobPublicAPI;
 import org.apache.shardingsphere.data.pipeline.api.PipelineJobPublicAPIFactory;
 import org.apache.shardingsphere.infra.distsql.update.RALUpdater;
-import org.apache.shardingsphere.migration.distsql.statement.DropMigrationStatement;
+import org.apache.shardingsphere.migration.distsql.statement.CleanMigrationStatement;
 
 /**
  * Drop migration updater.
```
Comment (yx9o) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/java/org/apache/shardingsphere/migration/distsql/handler/update/CleanMigrationUpdater.java:26]: Please update comments.

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/distsql/ral/impl/migration/update/CleanMigrationStatementAssert.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
 /**
  * Drop migration statement assert.
```
Comment (yx9o) [inline shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/statement/distsql/ral/impl/migration/update/CleanMigrationStatementAssert.java:32]: Please update comments.

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/distsql/ral/migration/CleanMigrationStatementTestCase.java [hunk 3]
```diff
@@ -28,7 +28,7 @@
  * Drop migration statement test case.
```
Comment (yx9o) [inline shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/distsql/ral/migration/CleanMigrationStatementTestCase.java:28]: Please update comments.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/antlr4/imports/migration/RALStatement.g4 [hunk 4]
```diff
@@ -39,8 +39,8 @@ stopScaling
     : STOP SCALING jobId
     ;
 
-dropScaling
-    : DROP SCALING jobId
+cleanScaling
+    : CLEAN SCALING jobId
```
Comment (yx9o) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-parser/src/main/antlr4/imports/migration/RALStatement.g4:43]: Please add `CLEAN` in the `Keyword.g4`.

**Comment by sandynz** [comment]
Hi @huangdx0726 , there's SQLParsingException of `clean-scaling`.
You could install `ANTLR v4` plugin (IDEA for example).
Open `migration/RALStatement.g4`; Select `cleanScaling`, select `Run Rule cleanScaling` in right click menu; Input test DistSQL; Then you could find whether DistSQL parsing work or not.
图片

**Review by sandynz — APPROVED** [review]
Thanks

**Pull request merged**